downstream/modules/aap-on-azure/proc-azure-hub-spoke-peering.adoc

```diff
@@ -0,0 +1,19 @@
+////
+Base the file name and the ID on the module title. For example:
+* file name: con-my-concept-module-a.adoc
+* ID: [id="con-my-concept-module-a_{context}"]
+* Title: = My concept module A
+////
+
+[id="proc-azure-hub-spoke-peering"]
+
+= Procedure
```

Comment:
I suggest removing the numbering for the Procedure heading. 
IBM Style Guide _Headings | Numbering_, also under _Procedures_ section: Generally, do not number headings, other than chapter titles.